pdf/pdfdocument.cpp

```diff
@@ -119,9 +125,18 @@ void PDFDocument::setSource(const QString& source)
     }
 }
 
+void PDFDocument::requestUnLock(const QString& password)
+{
+  if (!isLocked())
+      return;
+
+  UnLockDocumentJob* job = new UnLockDocumentJob{ password };
```

Comment:
I've used normal braces in object creation and removed unnecessary white spaces. I've also corrected some wrong indentation levels in other places of the patch.